client/nar/brainsimulation.py

```diff
@@ -109,24 +115,44 @@ def save(self, client, exists_ok=True):
         else:
             data["dateCreated"] = self.date_created
         if self.organization is not None:
-            if self.organization.id is None:
-                self.organization.save(client)
-            data["organization"] = {
-                "@type": self.organization.type,
-                "@id": self.organization.id
-            }
+            data["organization"] = [
+                {
+                    "@type": org.type,
+                    "@id": org.id,
+                } for org in self.organization
+             ]
         if self.PLA_components is not None:
             data["PLAComponents"] = self.PLA_components
         if self.alias is not None:
             data["alias"] = self.alias
         if self.model_of is not None:
             data["modelOf"] = self.model_of
         if self.brain_region is not None:
-            data["brainRegion"] = self.brain_region.to_jsonld()
+            data["brainRegion"] = [
```

Comment:
what is the reason for this change?

Reply:
for now it is not mandatory for the data migration but we will offer the possibility to add several brains regions, species etc... to a model project. I thought it would be better to add the possibility to save all the parameters as a list now, so we will not have to change the code after.

Reply:
ok, but then it should be `data["brainRegion"] = [br.to_jsonld() for br in self.brain_regions]` - i.e. keep the serialization in the BrainRegion class, within the `to_jsonld()` method. This ensures we don't have different serializations in different places.

Reply:
Just waiting for this to be fixed, then I can merge.